contracts/test/ProxyCustomLogic.sol

```diff
@@ -34,7 +34,7 @@ contract ProxyCustomLogic is IWalletCustomLogic {
         require(success, "call failed");
     }
 
-    function directExecute(address to, bytes calldata data) override external payable returns (
+    function directExecute(address to, uint256 value, bytes calldata data) override external payable returns (
         bytes memory ret  
     ) {  
         emit LogicCalled();  
```

Comment:
I think we should change line#42 to replace `msg.value` with `value` so it would be something like:
```
(success, ret) = to.call{value: value}(data);
```